plugins/rikki/heroeslounge/controllers/season/config_relation.yaml

```diff
@@ -39,3 +39,5 @@ teams:
         toolbarPartial: teams_toolbar
     manage:
         showSearch: true
+        recordsPerPage: 20
+        list: $/rikki/heroeslounge/models/team/small_list.yaml
```

Comment:
Please end files with a newline, it slightly messes up git history once anybody else adds more lines otherwise